pom.xml

```diff
@@ -29,6 +29,7 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <appengine.version>my_version_arafat</appengine.version>
```

Comment:
This should be commented out to be able to update to our official version on appengine.